docs/decisions/characteristics/0003-security.md

```diff
@@ -64,6 +64,9 @@ Configuration as Code will be used and run through static analysis to ensure onl
 ### Cross container communication of PII
 
 Static analysis will be run against cross-container communication contracts to ensure PII fields are not included.
+This includes Observability Signal egress, which can contain arbitrary data in attributes. In this case static analysis will be run against code to check for instrumentation that writes PII and additionally processors will be run in Telemetry to identify and redact potential PII prior to egress, and also alert on it.
+
+Finally, automated end-to-end system tests will be written covering all core flows beginning with registration. Test PII data used in registration will be checke for after each scenario in logs, metrics, traces, events and other containers stores. Finding the test PII Data outside the compartmentalized store fails the fitness function.
```

Comment:
check spelling

Reply:
thanks, fixing

Reply:
Should we talk about Data at rest as well?

Reply:
@kapooruma sure, do you have any ideas for fitness functions for data at rest?